Immense.RemoteControl.Desktop.Shared/Services/DesktopHubConnection.cs

```diff
@@ -184,6 +191,50 @@ public Task DisconnectViewer(IViewer viewer, bool notifyViewer)
         return Connection.SendAsync("DisconnectViewer", viewer.ViewerConnectionId, notifyViewer);
     }
 
+    public async Task GetScreenCast(
+        string viewerId,
+        string requesterName,
+        bool notifyUser,
+        bool enforceAttendedAccess,
+        string organizationName,
+        Guid streamId)
+    {
+        try
+        {
+            if (enforceAttendedAccess)
+            {
+                await SendMessageToViewer(viewerId, "Asking user for permission");
+
+                var result = await _remoteControlAccessService.PromptForAccess(requesterName, organizationName);
+
+                if (!result)
+                {
+                    await SendConnectionRequestDenied(viewerId);
+                    return;
+                }
+            }
+
+            // We don't want to tie up the invocation from the server, so we'll
+            // start this in a new task.
+            _ = Task.Run(async () =>
```

Comment:
No, since I'm using a discard to "fire and forget".  Since this is invoked from the server, I don't want the server to wait for the session to end (which is when the task will be completed).

A more "correct" way would be to push this to queue for processing, where the processor observes the task/session.  But, eh...